src/components/NavigationMenu/NavigationMenu.tsx

```diff
@@ -0,0 +1,76 @@
+import { AppBar, Box, Button, IconButton, Menu, MenuItem, Toolbar } from '@mui/material';
+
+import MenuIcon from '@mui/icons-material/Menu';
+import React from 'react';
+import styles from './NavigationMenu.module.scss';
+
+export default function NavigationMenu() {
+    const [anchorElNav, setAnchorElNav] = React.useState<null | HTMLElement>(null);
+    const navLinks = ['Home', 'Skills', 'Projects', 'Experience', 'Contact Me'];
+
+    const handleOpenNavMenu = (event: React.MouseEvent<HTMLElement>) => {
+        setAnchorElNav(event.currentTarget);
+    };
+
+    const handleCloseNavMenu = () => {
+        setAnchorElNav(null);
+    };
+
+    const handleNavLinkClick = () => {
+        handleCloseNavMenu();
+
+        // Navigate to selected page
+    };
+
+    return (
+        <AppBar className={styles.navigation_appbar}>
+            <Toolbar disableGutters className={styles.header_toolbar}>
+                {/* Collapsed Navigation Hamburger Menu for Smaller Screens */}
+                <Box sx={{ flexGrow: 1, display: { xs: 'flex', md: 'none' } }}>
```

Comment:
Love this! We could also use media query css to do this

Reply:
We don't have to change it, but I wanted to point that out

Reply:
Yeah, I built this largely basing off an example straight from the React Material UI documentation. So, this is how they seem to like to do media queries. I'm fine either way. I think media queries are easier to comprehend, but this seems more flexible.